src/Aspire.Dashboard/Components/Pages/Metrics.razor.cs

```diff
@@ -95,7 +99,7 @@ protected override async Task OnInitializedAsync()
             SelectedApplication = _selectApplication,
             SelectedDuration = _durations.Single(d => d.Id == s_defaultDuration),
             SelectedViewKind = null,
-            Dashpages = await DashpagePersistence.GetDashpagesAsync(CancellationToken.None)
+            Dashpages = []
```

Comment:
note: previous code fails on first invocation! as ViewModel will be null on first render